usecase/usecase-wrt-linux-tests/samples/CSP/README.md

```diff
@@ -0,0 +1,3 @@
+## Usecase Design
+
+This sample demonstrates the support for setting csp policies in manifest.json.
```

Comment:
s/csp/CSP/g

Because the CSP (content security policy) are mentioned for `manifest-src`, `default-src` for the manifest files, and `image-src` for icons at

- http://w3c.github.io/manifest/#content-security-policy
- http://w3c.github.io/manifest/#content-security-policy-of-image-objects

... let's focus on these checking.